### PR TITLE
Add Rome as a past event

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: _past_events/5-2018-rome.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+city: Rome
+year: 2018
+
+link: http://rome.rustfest.eu
```
Comment (apiraino) [inline _past_events/5-2018-rome.md]:
```suggestion
link: https://rome.rustfest.eu
```

File: _past_events/5-2018-rome.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - Mozilla
+  - Parity
+  - Baidu X-Lab
+  - Kodebox
+  - Threema
+  - Mullvad
+  - openSUSE
+  - rrbone
+  - PingCap
+---
```
Comment (apiraino) [inline _past_events/5-2018-rome.md:20]: Maybe add a link to the videos? Like:
```
videos: https://media.ccc.de/b/conferences/rustfest/2018-Rome
```

**Review by apiraino — APPROVED** [review]

File: _past_events/5-2018-rome.md (hunk 2 above)
Comment (badboy) [reply _past_events/5-2018-rome.md:20]: The videos attribute is not used anywhere yet (and we will publicly announce the video later this week)

Comment (apiraino) [reply _past_events/5-2018-rome.md:20]: oh ok I had just noticed that in the [Paris conf](https://github.com/RustFestEU/blog.rustfest.eu/blob/gh-pages/_past_events/4-2018-paris.md). But, yeah, it make sense to postpone until needed.

**Pull request merged**